module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPasswordCredential.md

```diff
@@ -55,23 +57,35 @@ This command creates new password credential for specified application.
 ### Example 2: Create a password credential using CustomKeyIdentifier parameter
 
 ```powershell
-New-EntraBetaApplicationPasswordCredential -ObjectId 'tttttttt-0000-2222-0000-aaaaaaaaaaaa' -CustomKeyIdentifier 'demoPassword'
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+$params = @{
+    ObjectId = 'tttttttt-0000-2222-0000-aaaaaaaaaaaa'
+    CustomKeyIdentifier = 'demoPassword'
+}
+
+New-EntraBetaApplicationPasswordCredential @params
```

Comment:
In example description we need t to add used parameters description for ObjectId and CustomKeyIdentifier